src/Client.php

```diff
@@ -161,29 +194,60 @@ private function post($route = '', array $body = [])
     }
 
     /**
-     * Perform cURL DELETE request to route. Used by
+     * Perform cURL GET request to route. For example used by
      * Alerts API requests: https://www.stathat.com/manual/alerts_api
      *
-     * @param string $route
-     * Ex: /x/ACCESSTOKEN/alerts/ALERTID
+     * @param string $curl_url
+     *   Ex: https://www.stathat.com/x/ACCESSTOKEN/alerts
+     * @return array or string $result
+     *   An array of alert details or the response code string in the case of a non 200 response.
      */
-    private function deleteAlertCurl($route = '')
+    private function curlGet($curl_url)
     {
-        $curlUrl = $this->alerts_url.'/'.$route;
-
         $ch = curl_init();
-        curl_setopt($ch, CURLOPT_URL, $curlUrl);
-        curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'DELETE');
-        curl_setopt($ch, CURLOPT_HEADER, true);
+        curl_setopt($ch, CURLOPT_URL, $curl_url);
         curl_setopt($ch, CURLOPT_HTTPHEADER, [
           'Accept: application/json',
           'Content-Type: application/json',
         ]);
         curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
-        curl_exec($ch);
-        $result = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+        $jsonResult = curl_exec($ch);
+        $result = json_decode($jsonResult);
+        $responseCode = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+        if ($responseCode != 200) {
+            $result = $responseCode;
```

Comment:
Since this seems "exceptional" to me, I think it would make sense to throw an `Exception` here with details on the error rather than requiring the user to check the type returned.